libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/components/avatar/AvatarSize.kt

```diff
@@ -18,11 +18,15 @@ package io.element.android.libraries.designsystem.components.avatar
 
 import androidx.compose.ui.unit.dp
 
-enum class AvatarSize(val value: Int) {
-    SMALL(32),
-    MEDIUM(40),
-    BIG(48),
-    HUGE(96);
+sealed class AvatarSize(open val value: Int) {
```

Comment:
What do you think of updating the type of the value to `Dp`? I think it would make things clearer when expecially using the `Custom` class.

Reply:
Yeah the change makes sense